.gitignore

```diff
@@ -95,3 +95,16 @@ commit-*
 *.swp
 *.patch
 *.exp
+
+# Qt Creator caches and settings
+build-*
+*.user*
+
+# CMake build files
+build/
+CMakeFiles/
+CMakeCache.txt
+cmake_install.cmake
+libxmp*.so*
```

Comment:
These three are not needed - already covered at top of the file

Reply:
I added these as they aren't actually ignored, and my git client showing them up. Lemme check the text...  That's possible there is a mistake above that didn't work for me, lemme me check...

Reply:
Ah, as far as I can see only `*.dylib` is missing, which needs adding among others at the top of `.gitignore`

Reply:
about `*.so`, it ignores only symlinks, but it doesn't ignores "*.so.x.x.x", so, I should replace:
```
*.so
```
with
```
*.so*
```
and problem solved

Reply:
OK, I just pushed a commit for `*.so*` and `*.dylib*` - please remember to rebase.